bezier-rs/lib/src/lib.rs

```diff
@@ -453,6 +453,46 @@ impl Bezier {
 			.try_into()
 			.unwrap()
 	}
+
+	/// Returns a list of lists of points representing the hull points for all iterations at the point corresponding to `t` using de Casteljau's algorithm
+	/// the ith list represents the points of the ith iteration
+	/// https://pomax.github.io/bezierinfo/#decasteljau
```

Comment:
Can you change the last line to say something like: `More information on the algorithm can be found in the [de Casteljau section in Pomax's Bezier Primer](https://pomax.github.io/bezierinfo/#decasteljau).`

Also btw, for urls, you can add `<url-here>` around it in the markdown so that rustdoc renders it as a hyperlink.

Reply:
Changed!